service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java

```diff
@@ -253,6 +253,9 @@ public static void executeOnEachServiceRegistry(Consumer<ServiceRegistry> action
 
   public static void addExtraServiceRegistry(ServiceRegistry serviceRegistry) {
     Objects.requireNonNull(serviceRegistry);
+    if (EXTRA_SERVICE_REGISTRIES.containsKey(serviceRegistry.getName())) {
```

Comment:
and name can not be the default regitry name

Reply:
getName() here returns custom registry name, not default name